subsys/bluetooth/controller/Kconfig

```diff
@@ -513,7 +513,9 @@ config BT_CTLR_SMI_TX_SETTING
 	help
 	  Enable support for Bluetooth 5.0 SMI TX through a system setting.
 
+source "subsys/bluetooth/controller/Kconfig.df"
 source "subsys/bluetooth/controller/Kconfig.ll_sw_split"
+source "subsys/bluetooth/controller/Kconfig.df"
```

Comment:
These lines here look kind of strange, but it works. I don't see how to make it more clean other other than moving `_SUPPORT` configs out of the `Kconfig.df` and to the top of `Kconfig`

Reply:
Oh... sorry left over. I'll clean it.